package-dev/Runtime/SentryInitialization.cs

```diff
@@ -25,6 +28,10 @@ namespace Sentry.Unity
 {
     public static class SentryInitialization
     {
+#if SENTRY_NATIVE_WINDOWS
```

Comment:
I wonder if it's best to move this into the package and just pass the "windows native" flag in. Like we did il2cpp. Goal is to keep this entry point free of logic and be able to test stuff in the test project (without the need of running this inot

Reply:
moved... should I also try to add a test case even though this is temporary until dotnet is fixed?

Reply:
yeah, at least now we know Mutex wouldn't work either if we're trying to add to .NET